.ci/build_msvc.bat

```diff
@@ -1,19 +1,34 @@
 
 call "C:\Program Files (x86)\Microsoft Visual Studio 14.0\VC\vcvarsall.bat" amd64
-:: For gperf
-C:\msys64\usr\bin\bash -lc "pacman --verbose --noconfirm -Su" || goto :error
-C:\msys64\usr\bin\bash -lc "pacman --verbose --noconfirm --needed -S gperf" || goto :error
 
+set CMAKE_GENERATOR="Visual Studio 14 2015 Win64"
```

Comment:
My olny reason for using nmake is that it is usually more verbose on errors. With the VS generator the vcvarsall line can be removed, I think.